tests/foreman/maintain/test_backup_restore.py

```diff
@@ -254,23 +274,29 @@ def test_positive_backup_offline_logical(sat_maintain, setup_backup_tests, modul
     assert 'FAIL' not in result.stdout
 
     # Check for expected files
-    backup_dir = re.findall(fr'{subdir}\/satellite-backup-.*-[0-5][0-9]', result.stdout)[0]
+    backup_dir = re.findall(fr'{subdir}\/{instance}-backup-.*-[0-5][0-9]', result.stdout)[0]
     files = sat_maintain.execute(f'ls -a {backup_dir}').stdout.split('\n')
     files = [i for i in files if not re.compile(r'^\.*$').search(i)]
 
-    if sat_maintain.is_remote_db():
-        expected_files = BASIC_FILES | REMOTE_SAT_FILES
+    if type(sat_maintain) is Satellite:
```

Comment:
is there a reason not to use `get_expected_files` also here

Reply:
yes, in this particular case we expect files for both backup types created